brat/Brat/Compile/Hugr.hs

```diff
@@ -201,7 +200,7 @@ compileConst parent tm ty = do
   constId <- addNode "Const" (OpConst (ConstOp parent (valFromSimple tm)))
   loadId <- case ty of
     HTFunc poly@(PolyFuncType [] _) ->
-      addNode "LoadFunction" (OpLoadFunction (LoadFunctionOp parent poly [] (FunctionType [] [HTFunc poly])))
+      addNode "LoadFunction" (OpLoadFunction (LoadFunctionOp parent poly [] (FunctionType [] [HTFunc poly] [])))
```

Comment:
Why are the extension reqs for `LoadFunction` empty?

Reply:
LoadFunction is only used for const functions i thought? The extensions in the function that's loaded are boxed up until it is `call`ed

Reply:
I have no idea, just checking 😅

But that makes sense 👍